microcloud/test/includes/microcloud.sh

```diff
@@ -849,8 +849,8 @@ setup_lxd_project() {
     fi
 
     lxc remote switch local
-	  lxc project create microcloud-test || true
-	  lxc project switch microcloud-test
+	lxc project create microcloud-test || true
```

Comment:
why `|| true` here?

Reply:
If the project doesn't exist, it will create the project.

If the project does exist, it will won't break at trying to create the project and will continue with the test. This is very helpful when you're running tests locally and something breaks so the tests don't clean up properly.

Reply:
hrm yeah but this could be applied to any state setup not just projects.
Seemed kind of odd to see this here as it suggests we dont know what state the host will be in when the tests are running.

Reply:
The only danger I can see is that if a user had a pre-existing `microcloud-test` project, then their default profile in that project is going to get wiped by running the test suite. That and maybe pre-existing project restrictions but that would cause the test suite to fail the initial setup anyway. 

As for the state of the hosts, it's inconsequential to the tests which other networks, profiles, storage pools, or instances exist on that project and we're strict about the creation of any of those that are necessary for the test suite. In the case of CI, since the whole LXD is replaced per run, then it doesn't matter for that either.

I think it's useful to have this for making it easy to debug the test suite but I'm fine adding it manually if I need it locally.